### PR TITLE
Ignore `exact` keywords for binary operations.

### PR DESCRIPTION
**Pull request by ptersilie**
Until we start optimising traces we don't need to worry about them.

### CODE REVIEW

**Comment by ptersilie** [comment]
There's a `yk` PR coming, but it'll have to wait until your PRs have gone in as there will be merge conflicts.

File: llvm/lib/YkIR/YkIRWriter.cpp [hunk 1]
```diff
@@ -373,19 +373,17 @@ class YkIRWriter {
...
       return;
     }
 
     // Note that we do nothing with the `nsw` and `nuw` (no {signed,unsigned}
-    // wrap) keywords, which may generate poison values. If they do, the rules
-    // of deferred UB allow us to make any value we wish, including (as
-    // we do) the wrapped value.
+    // wrap), and `exact` keywords, which may generate poison values. If they
+    // do, the rules of deferred UB allow us to make any value we wish,
+    // including (as we do) the wrapped value.
```
[the remark this answers is not in the document]
Comment (ptersilie) [reply llvm/lib/YkIR/YkIRWriter.cpp:386]: I've already rebased my local branch and don't want to untangle that again. Can I do this while rebasing?

[the remark this answers is not in the document]
Comment (vext01) [reply llvm/lib/YkIR/YkIRWriter.cpp:386]: yep, that's fine.

**Comment by vext01** [comment]
Do you want to rebase then?

**Comment by vext01** [comment]
Sorry, must have clicked the wrong button there!

**Comment by ptersilie** [comment]
Rebased and fixed the comment.

**Pull request merged**